tests/extension/khr_queue_flush/CMakeLists.txt

```
@@ -0,0 +1,5 @@
+if(SYCL_CTS_ENABLE_QUEUE_FLUSH_TESTS)
```

```suggestion
if(SYCL_CTS_ENABLE_KHR_QUEUE_FLUSH_TESTS)
```
Please, add option to the root CMakeLists.txt. E.g. https://github.com/KhronosGroup/SYCL-CTS/blob/38321160c6d2986fc09a3720a1bbf38d6541c6a2/CMakeLists.txt#L147-L149

---

For `SYCL_CTS_ENABLE_QUEUE_FLUSH_TESTS`, good point.  Then I should change that also in the `queue empty one`.

Sorry for root CMakelist

---

Fixed with 3b4e8a1d875a7e715e9b631af3c61fdde9a5595a